server/src/main.rs

```diff
@@ -283,22 +140,32 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 std::process::exit(1);
             }
         },
-        Command::Uninstall { interface, yes } => uninstall(&interface, &conf, opts.network, yes)?,
+        Command::Uninstall { interface, yes } => {
+            innernet_server::uninstall(&interface, &conf, opts.network, yes)?
```

Comment:
Let's import items from `innernet_server` instead of qualifying them. It will make the diff smaller (and lower the fallout from renaming to `innernet_server`).